bin/yb-ctl

```diff
@@ -533,6 +540,7 @@ class ClusterOptions:
         self.enable_ysql = None
         self.install_if_needed = False
         self.yb_ctl_verbose = False
+        self.cluster_config = None
```

Comment:
can we group the fields that are saved to/from the config here somehow together?